crates/ruma-client-api/src/authenticated_media/get_content_thumbnail.rs

```diff
@@ -70,18 +70,12 @@ pub mod v1 {
 
         /// Whether the server should return an animated thumbnail.
         ///
-        /// When `true`, the server should return an animated thumbnail if possible and supported.
-        /// Otherwise it must not return an animated thumbnail.
-        ///
-        /// Defaults to `false`.
-        #[cfg(feature = "unstable-msc2705")]
+        /// When `Some(true)`, the server should return an animated thumbnail if possible and
+        /// supported. When `Some(false)`, the server must not return an animated
+        /// thumbnail. When `None`, the server should not return an animated thumbnail.
         #[ruma_api(query)]
-        #[serde(
-            rename = "org.matrix.msc2705.animated",
-            default,
-            skip_serializing_if = "ruma_common::serde::is_default"
-        )]
```

Comment:
It might cause issues. In the past clients have sent both the stable and unstable name, so serde returns an error because it considers that the same field is present twice.

See #1266.